spec/merch_calendar/date_calculator_spec.rb

```diff
@@ -48,5 +48,43 @@
 
   it "#end_of_quarter"
 
+  describe "#merch_months_in" do
+    it "returns merch date for start_date if start_date is the same as end_date" do
+      start_date = Date.new(2014,8,1)
+      end_date = start_date
+      start_merch_date = MerchCalendar.start_of_month(start_date.year, merch_month: start_date.month)
+
+      merch_months = subject.merch_months_in(start_date, end_date)
+      expect(merch_months.count).to be(1)
+      expect(merch_months.first.year).to eq start_merch_date.year
+      expect(merch_months.first.month).to eq start_merch_date.month
+      expect(merch_months.first.day).to eq start_merch_date.day
+    end
+
+    it "returns valid merch dates for 2014" do
+      start_date = Date.new(2014, 1, 1)
```

Comment:
Is the expectation that `merch_months_in` will return the next merch months with a start date on or after this start date? I.e. in this example, the first merch month it returns will be the first merch month that starts on or after 1/1/2014?